appserver/ejb/ejb-connector/src/main/java/org/glassfish/ejb/config/MdbContainer.java

```diff
@@ -73,8 +73,8 @@ public interface MdbContainer extends ConfigBeanProxy, PropertyBag, ConfigExtens
      * @return possible object is
      *         {@link String }
      */
-    @Attribute (defaultValue="0")
-    @Min(value=0)
+    @Attribute(defaultValue = "1")
```

Comment:
Is this another case of fixing a _ProcessInjectionTarget_ event not being fired by eagerly loading the thing it should be fired for?
If so this isn't a fix, it's a bit of a hack to pass the TCK and I think we should instead be looking at properly fixing the hole we have in that we're not firing the events for things which haven't been instantiated yet.

> "The container must also fire an event for every Jakarta EE component class supporting injection that ----> may <------ be instantiated by the container at runtime".

Reply:
This also strikes me as something we don't necessarily have to commit to the code, and if we did want this hack we could just include it as pre-test setup.

Reply:
I removed those changes and I followed same strategy you did for the servlet to fire the event. Now if the MDB has an injection the event is fired causing to pass the tck test ContainerEventTest:testProcessInjectionTargetEventFiredForMessageDrivenBean